megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -975,7 +975,12 @@ public void actionPerformed(ActionEvent event) {
             case VIEW_MOVE_ENV:
                 if (curPanel instanceof MovementDisplay) {
                     GUIP.setMoveEnvelope(!GUIP.getMoveEnvelope());
-                    ((MovementDisplay) curPanel).computeMovementEnvelope(getUnitDisplay().getCurrentEntity());
+                    Entity entity = getUnitDisplay().getCurrentEntity();
+                    if (!(entity instanceof Aero)) {
```

Comment:
`entity instanceof Aero` is false for LAMs in fighter mode.
```suggestion
                    if (!entity.isAero()) {
```